packages/text-link/src/TextLink.test.js

```diff
@@ -47,4 +47,21 @@ describe("TextLink", () => {
       ).toHaveProp("target", "_parent");
     });
   });
+
+  describe("the element rendered", () => {
+    describe("given a link", () => {
+      it("it is an <a> tag", () => {
+        expect(
+          mount(
+            <TextLink link="https://www.autodesk.com/">Link</TextLink>
+          ).find("a")
+        ).toHaveLength(1);
+      });
+    });
+    describe("if link is not given", () => {
+      it("it is a <span> tag", () => {
+        expect(mount(<TextLink>Link</TextLink>).find("span")).toHaveLength(1);
```

Comment:
expect(
  mount(
    <TextLink link="https://www.autodesk.com/">Link</TextLink>
  ).toContain('span');
);

Reply:
I tried using `toContain` and didn't seem to work as expected, as it through back an error. In looking at the error it was trying to match the `value` I passed into the mounted `object` and wasn't able to match just `a` or `span`. I also tried passing in the entire `node` that was in the messaging and it still didn't pass.

I think the way I currently have it while not as readable might be a more suitable test since it is trying to find the selector within the mounted wrapper.